Dockerfile.windows

```diff
@@ -0,0 +1,5 @@
+# for testing purposes
+# checkov:skip=CKV_DOCKER_3:Suppress error "Ensure that a user for the container has been created" since this Dockerfile is only used for testing.
```

Comment:
Can we configure Checkov to ignore this error in all Dockerfiles so that we don't have to duplicate the suppression?